tests/modules/eido/validate/main.nf

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { EIDO_VALIDATE } from '../../../../modules/eido/validate/main.nf'
+
+workflow test_eido_validate_on_nextflow_samplesheet {
+
+    samplesheet = file(params.test_data['pep']['nextflow_samplesheet'], checkIfExists: true)
+    schema = file(params.test_data['pep']['schema'], checkIfExists: true)
+
+    EIDO_VALIDATE ( samplesheet, schema )
+}
+
+workflow test_eido_validate_on_pep_config {
+
+    samplesheet = file(params.test_data['pep']['pep_config'], checkIfExists: true)
```

Comment:
```suggestion
    samplesheet = file("https://github.com/nf-core/test-datasets/blob/modules/data/delete_me/pep/test_pep_format_files/config.yaml", checkIfExists: true)
```

Reply:
Does the `sample_table` in the config.yaml need to point to a remote file in this case? Currently the samplesheet.csv is not loaded into the working directory itself, so I suspect this wouldn't work(?)